pkg/virt-controller/watch/vm_test.go

```diff
@@ -5531,7 +5539,7 @@ var _ = Describe("VirtualMachine", func() {
 				expectControllerRevisionList()
 				expectControllerRevisionCreation()
 				controller.Execute()
-				Expect(cache.WaitForCacheSync(stop, crInformer.HasSynced)).To(BeTrue())
+				syncCaches()
```

Comment:
I am more or less convinced this is no-op. 
```
// HasSynced returns true if the shared informer's store has been
// informed by at least one full LIST of the authoritative state
// of the informer's object collection.  This is unrelated to "resync".
```

Reply:
Yeah, you're *probably* right... But, as we now know, things need time to settle after controller.Execute(), which also doesn't seem to makes sense...